php/login.php

```diff
@@ -0,0 +1,141 @@
+    }
+    unset($stmt);
+  }
+  unset($conn);
+}
+?>
+
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+  <link rel="stylesheet" href="../css/login.css">
+  <script src="../js/login.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+  <title>Login</title>
+</head>
+
+<body>
+  <script>
+    if (window.localStorage["registered"] == "true") {
+      let toastHTMLElement = document.getElementById("email_duplicate_toast");
+      document.getElementById("error_text").textContent = "data.errorText;"
+      let toastElement = new bootstrap.Toast(toastHTMLElement);
+      toastElement.show();
+      localStorage.clear();
+    }
+  </script>
+  <div class="login-image"></div>
+  <div aria-live="polite" aria-atomic="true" class="d-flex justify-content-center align-items-center m-2 toasts">
+    <div class="toast-container">
+      <div class="toast rounded" role="alert" aria-live="assertive" aria-atomic="true" id="email_duplicate_toast">
+        <div class="d-flex alert-success rounded">
+          <div class="toast-body">
+            <b>Success!</b> You have been successfully registered
+          </div>
+          <button type="button" class="btn-close me-2 m-auto" data-bs-dismiss="toast" aria-label="Close"></button>
+        </div>
+      </div>
+
+      <div class="toast align-items-center" role="alert" aria-live="assertive" aria-atomic="true" id="error_toast">
+        <div class="d-flex alert-danger rounded">
+          <div class="toast-body">
+            <b>Something went wrong!</b><br><span id="error_text"></span>
+          </div>
+          <button type="button" class="btn-close me-2 m-auto" data-bs-dismiss="toast" aria-label="Close"></button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <main class="login-form">
+    <form action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]); ?>" method="POST">
+      <h1 class="h3 mb-3 fw-normal">Please log in</h1>
+      <input type="email" class="form-control" name="email" placeholder="name@example.com" required><br>
+      <input type="password" class="form-control" name="password" placeholder="Password" id="pwd" required>
+      <span class="show-hide-pwd-eye" onclick="show_hide_pwd()">
+        <img src="../img/eye.svg">
+      </span>
+      <div class="checkbox mb-3 align-center">
+        <label>
+          <input type="checkbox" value="remember-me"> Remember me
+        </label>
+      </div>
+      <button class="w-100 btn btn-lg btn-primary" type="submit">Login</button><br><br>
```

Comment:
<br> shouldn't be needed.

Reply:
Please clearify what shouldnt be needed. The button?

Reply:
The br at the end. I think my comment above is bugged 😆